frontend/interfaces/EventTypes.ts

```diff
@@ -26,7 +36,8 @@ interface AbstractEventData {
   eventTags: string[]; // Assuming "list of tags" is an array of strings
   isActive: boolean;
   isPrivate: boolean;
-  attendees: { email: string }[];
+  attendees: EventAttendees;
+  attendeesMetadata: EventAttendeesMetadata;
   accessCount: number;
   sport: string;
```

Comment:
do we need image: string; ?

Reply:
do we not? thought it was a link to image from firebase directory - we have the same implementation for profile picture